splitio/sync/synchronizer.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, split_sync, segment_sync, impressions_sync, events_sync,  # p
         self._events_sync = events_sync
         self._impressions_count_sync = impressions_count_sync
 
+    def set_none_sync(self, unique_keys_sync, clear_filter_sync):
```

Comment:
if you allow this sync in init as None in default, you can avoid this method as you did in Tasks